### ISSUE TITLE
Потребление RAM

### ISSUE THREAD

**Issue by demos98** [KitKatDen/mvideo_test#1]
Результат не удовлетворяет условиям.
Используется  больше 5GB RAM.

<img width="535" alt="Screen Shot 2021-06-08 at 01 38 48" src="https://user-images.githubusercontent.com/10990163/121091264-ca04dd80-c7fa-11eb-8453-d852f92985a6.png">

**Comment by KitKatDen** [comment]
Да, во время обзора моего тестового задания выяснял это с собеседующим, надо докрутить
